src/tests/src/solver/optimisation/adequacy_patch.cpp

```diff
@@ -372,75 +371,3 @@ BOOST_AUTO_TEST_CASE(check_adq_param_wrong_hurdle_cost)
     auto p = createParams();
     BOOST_CHECK_THROW(p.checkAdqPatchIncludeHurdleCost(false), Error::IncompatibleHurdleCostCSR);
 }
-
-BOOST_AUTO_TEST_SUITE(adq_patch_post_processing)
```

Comment:
We totally removed the functions tested here, so associated tests were removed too.
These functions were judged doing too few things and lowering the clarity in the caller.